js/src/modules/_MobileMenu.es6.js

```diff
@@ -164,7 +164,14 @@ class _MobileMenu {
   _setTabIndex(elem, tabIndex) {
     if (Array.isArray(elem)) {
```

Comment:
it might be cleaner to either check if the parameter is iterable (array or NodeList) then convert it to an array instead of doing the for loop
```function isIterable(obj) {
  // checks for null and undefined
  if (obj == null) {
    return false;
  }
  return typeof obj[Symbol.iterator] === 'function';
}```

Reply:
> it might be cleaner to either check if the parameter is iterable (array or NodeList) then convert it to an array instead of doing the for loop
> 
> ```
> function isIterable(obj) {
>   // checks for null and undefined
>   if (obj == null) {
>     return false;
>   }
>   return typeof obj[Symbol.iterator] === 'function';
> }
> ```

@sbrack8t This JS is over my head, and I’m not sure how I would implement this. Is the potential performance improvement worth it?

Reply:
For the number of potential items in the menu list there's probably not much of a performance gain. I'm fine leaving it as is.